savant/utils/memory_repr_pytorch.py

```diff
@@ -0,0 +1,40 @@
+import cv2
+import torch
+
+from savant.utils.memory_repr import OpenCVGpuMatCudaArrayInterface, \
+    _numpy_type_to_opencv_mat_type
+
+
+__all__ = [
+    'opencv_gpu_mat_as_pytorch',
+    'pytorch_tensor_as_opencv_gpu_mat',
+]
```

Comment:
let's use one variant for naming, `pytorch_tensor` (opencv_gpu_mat_as_pytorch_tensor+ pytorch_tensor_as_opencv_gpu_mat) or `pytorch` (opencv_gpu_mat_as_pytorch + pytorch_as_opencv_gpu_mat)

I just used `cupy` and added `gpu_mat` for opencv because cupy has only one data representation - ndarray, and opencv has 2 - Mat and GpuMat. So I chose the full version for opencv to be clear.